docs/client-sdk/javascript/tutorials/filter-and-label-conversations-by-id-and-metadata.md

```diff
@@ -0,0 +1,177 @@
+
+This example sets `mydomain.xyz/alert` as the `conversationId`, indicating that these conversations are alerts. In addition, the example sets a conversation metadata `title` value to `P0`.
+
+You can then use this conversation ID and metadata to display only alerts created by your app with a severity of P0, for example.
+
+```js showLineNumbers
+// Start a scoped conversation with ID mydomain.xyz/alert and add some metadata
+const conversation2 = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897',
+  {
+    conversationId: 'mydomain.xyz/alert',
+    metadata: {
+      title: 'P0',
+    },
+  }
+)
+```
+
+
+## Filter conversations by IDs and metadata
+
+Now that you've set conversation IDs and metadata, you can use them to filter conversations in your app.
+
+<!--I move between using conversation ID and conversationId - maybe just stick to conversationId? Any preferences? JHA to look up dev doc style guidelines.-->
+
+1. Get all of the conversations, as shown in lines 1-2.
+
+2. Filter by the domain value in your `conversationId`, `mydomain.xyz/` for example, to return only conversations created by your app, as shown in lines 3-8.
+
+3. Further filter by the rest of the `conversationId` and metadata. For example, filter by `notif` to display only notification messages, as shown in lines 10-14. And separately filter by `alert` and display the metadata `title` value of `P0` in your app, as shown in lines 15-18.
+
+<!--correct about lines 15-18 displaying the metadata title value of P0 in the app? just to be sure I understand, do we want to surface that you can use metadata values to display useful info about a conversation in your UI? Should I add it to the heading for this section? Can I also use metadata as a filter - or is it more for displaying info in the UI?-->
+
+```js showLineNumbers
+// Get all the conversations
+const conversations = await xmtp.conversations.list()
+// Filter for conversations from your application
+const myAppConversations = conversations.filter(
+  (convo) =>
+    convo.context?.conversationId &&
+    convo.context.conversationId.startsWith('mydomain.xyz/')
+)
+
+for (const conversation of myAppConversations) {
+  const conversationId = conversation.context?.conversationId
+  if (conversationId === 'mydomain.xyz/notif') {
+    await conversation.send('notif')
+  }
+  if (conversationId === 'mydomain.xyz/alert') {
+    await conversation.send('alert')
+    console.log(conversation.context?.metadata.title)
+  }
+}
+```
+
+<!--I think I need to move the following section to a separate tutorial. The dev who needs the info below is kind of doing the opposite of the devs who need the info above. For example, the dev using the info below ISN'T filtering, but rather is adjusting the universal inbox in their app to accommodate the impact of other apps that are using conversation IDs to filter. Is this assessment correct?-->
```

Comment:
I could see this being its own document as well. You're right that the use-cases are very different.